docs/roman/includes/standard_keywords.inc

```diff
@@ -1,27 +1,25 @@
 
-Standard Keywords
+Standard ASDF metadata tags
 +++++++++++++++++
-The following table lists the keywords that are *required* to be present in all
-reference files.  The first column gives the keyword name.  The second column 
-gives the roman data model name for each keyword, which is useful when using
-data models in creating and populating a new reference file.  The third column
-gives the equivalent meta tag in ASDF reference file headers, which is the same
-as the name within the data model meta tree (second column).
+The following table lists the tags that are *required* to be present
+in all reference files.  The first column gives meta tag in ASDF
+reference file headers, which is the same as the name within the data
+model meta tree (second column).  The second column gives the roman
+data model name for each keyword, which is useful when using data
+models in creating and populating a new reference file.
 
-==========  ==========================  ========================
-Keyword     Data Model Name             ASDF meta tag
-==========  ==========================  ========================
-AUTHOR      model.meta.author           author
-DATAMODL    model.meta.model_type       model_type
-DATE        model.meta.date             date
-DESCRIP     model.meta.description      description
-FILENAME    model.meta.filename         N/A     
-INSTRUME    model.meta.instrument.name  instrument: {name}
-PEDIGREE    model.meta.pedigree         pedigree
-REFTYPE     model.meta.reftype          reftype
-TELESCOP    model.meta.telescope        telescope
-USEAFTER    model.meta.useafter         useafter
-==========  ==========================  ========================
+============  ========================
+ASDF meta tag    Data Model Name
```

Comment:
After talking it over with ASDF devs the suggestion is to use

`Attribute` (instead of `keyword` or `ASDF tag`)
`Fully qualified path in the Data Model structure` (instead of Data Model Name`)

Reply:
`Tag` in ASDF has a different meaning. It associates an in stance of a data type with the appropriate schema.

Reply:
I think I've updated the files as requested.